### ISSUE TITLE
Failed to install K8s 1.23 and containerd/docker

### ISSUE THREAD

**Issue by FeynmanZhou** [kubesphere/kubekey#933]
### What is version of KubeKey has the issue?

2.0.0 alpha 3

### What is your os environment?

CentOS 7.7

### KubeKey config file

No config file.

### A clear and concise description of what happend.

I installed Kubernetes 1.23 and containerd/docker using KubeKey alpha 3 release:

```
./kk create cluster --with-kubernetes v1.23.0 --container-manager contained
```

But I was failed to install it on my server with an all-in-one mode.

### Relevant log output

Install K8s 1.23 and containerd:

```shell
23:01:12 CST [InstallContainerModule] Sync crictl binaries
23:01:13 CST message: [i-a26jzcsm]
install crictl binaries failed: Failed to exec command: sudo -E /bin/bash -c "mkdir -p /usr/bin && tar -zxf /tmp/kubekey/crictl-v1.22.0-linux-amd64.tar.gz -C /usr/bin "
tar (child): /tmp/kubekey/crictl-v1.22.0-linux-amd64.tar.gz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now: Process exited with status 2
23:01:13 CST retry: [i-a26jzcsm]
23:01:18 CST message: [i-a26jzcsm]
install crictl binaries failed: Failed to exec command: sudo -E /bin/bash -c "mkdir -p /usr/bin && tar -zxf /tmp/kubekey/crictl-v1.22.0-linux-amd64.tar.gz -C /usr/bin "
tar (child): /tmp/kubekey/crictl-v1.22.0-linux-amd64.tar.gz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now: Process exited with status 2
23:01:18 CST retry: [i-a26jzcsm]
23:01:18 CST failed: [i-a26jzcsm]
error: Pipeline[CreateClusterPipeline] execute failed: Module[InstallContainerModule] exec failed:
failed: [i-a26jzcsm] [SyncCrictlBinaries] exec failed after 2 retires: install crictl binaries failed: Failed to exec command: sudo -E /bin/bash -c "mkdir -p /usr/bin && tar -zxf /tmp/kubekey/crictl-v1.22.0-linux-amd64.tar.gz -C /usr/bin "
tar (child): /tmp/kubekey/crictl-v1.22.0-linux-amd64.tar.gz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now: Process exited with status 2
```

Install K8s 1.23 and docker:

```
3:02:51 CST [InstallContainerModule] Enable docker
23:02:56 CST success: [i-a26jzcsm]
23:02:56 CST [InstallContainerModule] Add auths to container runtime
23:02:56 CST message: [i-a26jzcsm]
copy docker auths failed cmd: mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json , err:Failed to exec command: sudo -E /bin/bash -c "mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json "
cp: cannot stat ‘/root/.docker/config.json’: No such file or directory: Process exited with status 1: Failed to exec command: sudo -E /bin/bash -c "mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json "
cp: cannot stat ‘/root/.docker/config.json’: No such file or directory: Process exited with status 1
23:02:56 CST retry: [i-a26jzcsm]
23:03:01 CST message: [i-a26jzcsm]
copy docker auths failed cmd: mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json , err:Failed to exec command: sudo -E /bin/bash -c "mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json "
cp: cannot stat ‘/root/.docker/config.json’: No such file or directory: Process exited with status 1: Failed to exec command: sudo -E /bin/bash -c "mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json "
cp: cannot stat ‘/root/.docker/config.json’: No such file or directory: Process exited with status 1
23:03:01 CST retry: [i-a26jzcsm]
23:03:06 CST message: [i-a26jzcsm]
copy docker auths failed cmd: mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json , err:Failed to exec command: sudo -E /bin/bash -c "mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json "
cp: cannot stat ‘/root/.docker/config.json’: No such file or directory: Process exited with status 1: Failed to exec command: sudo -E /bin/bash -c "mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json "
cp: cannot stat ‘/root/.docker/config.json’: No such file or directory: Process exited with status 1
23:03:06 CST retry: [i-a26jzcsm]
23:03:06 CST failed: [i-a26jzcsm]
error: Pipeline[CreateClusterPipeline] execute failed: Module[InstallContainerModule] exec failed:
failed: [i-a26jzcsm] [Login PrivateRegistry] exec failed after 3 retires: copy docker auths failed cmd: mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json , err:Failed to exec command: sudo -E /bin/bash -c "mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json "
cp: cannot stat ‘/root/.docker/config.json’: No such file or directory: Process exited with status 1: Failed to exec command: sudo -E /bin/bash -c "mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json "
cp: cannot stat ‘/root/.docker/config.json’: No such file or directory: Process exited with status 1
```

**Comment by 24sama** [comment]
Fix this issue by: #934, #936. And we will release a new alpha version soon.

**Comment by 24sama** [comment]
v2.0.0-alpha.4 has been released. This issue could be closed.
